.github/workflows/release-dataset.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish dataset to production DB
+
+on:
+  workflow_dispatch:
+    branches: 
+      - main
+    inputs: 
+      megaset-name:
+        description: "Enter the name of the megaset to release every dataset within"
+        required: false
+      contained-dataset-id: 
+        description: "Enter the ID in the format [NAME]_v[VERSION] to release a single dataset contained within a megaset"
+        required: false
+      standalone-dataset-id: 
+        description: "Enter the ID in the format [NAME]_v[VERSION] to release a single standalone dataset"
+        required: false
+      
+
+jobs: 
+  release_dataset: 
+    name: Release Dataset
+    runs-on: ubuntu-latest
+    env:
+      NODE_ENV: "production"
+      FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+      FIREBASE_EMAIL: ${{ secrets.FIREBASE_EMAIL }}
+      AWS_SECRET: ${{ secrets.CFE_AWS_SECRET }}
+      AWS_ID: ${{ secrets.CFE_AWS_ID }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - run: npm ci
+      - name: Run release-dataset script
+        run: |
+          if: ${{ github.event.inputs.megaset-name }} && ${{ github.event.inputs.contained-dataset-id }}
+            npm run release-dataset ${{ github.event.inputs.megaset-name }} ${{ github.event.inputs.contained-dataset-id }}
+          elif [ -n "${{ github.event.inputs.megaset-name }}" ]; then
```

Comment:
Is this elif in the action code? I don't remember seeing this as an available conditional.